src/tests/test_config.py

```diff
@@ -45,6 +45,7 @@ class TestProcessConfig(TestProcessProto):
     def test_config_defaults(self):
         """Test settings in the generated config"""
         self._stop_process()
+        self._kill_process()
```

Comment:
This is like `tearDown()` without cleaning the files.

Reply:
Yes I think you're correct.